apidoc/Titanium/UI/iOS/Stepper.yml

```diff
@@ -48,28 +53,35 @@ properties:
 
   - name: wraps
     summary: |
-        If YES, incrementing beyond maximumValue sets value to minimumValue; likewise, decrementing below
-        minimumValue sets value to maximumValue. If NO, the stepper does not increment beyond maximumValue nor
-        does it decrement below minimumValue but rather holds at those values.
+        If YES, incrementing beyond <Ti.UI.iOS.Stepper.maximum> sets value to <Ti.UI.iOS.Stepper.minimum>. likewise, decrementing below
+        <Ti.UI.iOS.Stepper.minimum> sets value to <Ti.UI.iOS.Stepper.maximum>. If NO, the stepper does not increment beyond <Ti.UI.iOS.Stepper.maximum> nor
```

Comment:
This is correct, think of it like a cycle if you keep going up and go past the maximum value. You start from the beginning. Its not a validation where if ,for example the value is less than 0 we set it to 0.